app/models/web_hook.rb

```diff
@@ -57,18 +61,16 @@ def self.enqueue_topic_hooks(event, topic, user)
   %i(post_created
      post_destroyed
      post_recovered).each do |event|
-
     DiscourseEvent.on(event) do |post, _, user|
-      WebHook.enqueue_hooks(:post,
-        post_id: post.id,
-        topic_id: post&.topic&.id,
-        user_id: user&.id,
-        category_id: post.topic&.category&.id,
-        event_name: event.to_s
-      )
+      WebHook.enqueue_post_hooks(event, post, user)
     end
   end
 
+  DiscourseEvent.on(:before_edit_post) do |post, topic_changed|
```

Comment:
Hmm sorry I just realized that this event is called before the transaction has been committed. We probably want to add event triggers for after the post has been committed to the DB